pkg/scraper/mapped.go

```diff
@@ -368,6 +395,11 @@ func (a mappedPostProcessAction) ToPostProcessAction() (postProcessAction, error
 		action := postProcessMap(a.Map)
 		ret = &action
 	}
+	if a.FeetToCm {
+		found = "toMetric"
```

Comment:
`found = "toMetric" ` after the refactoring shouldn't it be `feetToCm` ?

Reply:
Oops, missed that one.